src/Dapr.Client/InvocationHandler.cs

```diff
@@ -62,7 +54,10 @@ private static string GetDefaultDaprEndpoint()
         public InvocationHandler()
         {
             this.parsedEndpoint = new Uri(GetDefaultDaprEndpoint(), UriKind.Absolute);
-            this.apiToken = GetDaprApiToken();
+            if(string.IsNullOrWhiteSpace(this.apiToken))
+            {
+                this.apiToken = Environment.GetEnvironmentVariable("DAPR_API_TOKEN");
```

Comment:
Again, this logic shouldn't appear everywhere. It's OK to just make the existing property public